frontend/src/store/modules/penRequest.js

```diff
@@ -7,12 +11,52 @@ export default {
       legalLastName: '',
       legalFirstName: '',
       reviewer: '',
+      documentTypes: [],
```

Comment:
the PenRequestDetail uses `this.$store.state['penRequest'].documentTypes` to get document types, but the store defines the documentTypes as a part of headerSearchParams.